llama_index/graph_stores/nebulagraph.py

```diff
@@ -327,7 +344,6 @@ def get_flat_rel_map(
         subjs_ = result.column_values("subj") or []
```

Comment:
If the query keywords is not found in nebulagraph, the result type will be NoneType.
result.column_values will cause
AttributeError: 'NoneType' object has no attribute 'column_values'

Reply:
Thanks @BleakStone! I'll fix this in upcoming PRs, or if you are interested, you could help fix this, too!